cernopendata_client/helper.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```

Comment:
`helper.py` is a rather generic name... since it can help with anything.  Better invent some more targeted name. 

- One option is to put it in `download.py` and have all the functions related to downloads sitting in that file.
- Another option is to put it in `tui.py` and have all the "text user interface" functions sitting in that file.

The former is probably better, since we wouldn't be easily able to reuse this function elsewhere in other contexts than downloading.

WDYT?

Reply:
The reason why I choose `helper.py` is that I thought we will be adding other functions that are on their own and don't require any data apart from arguments and we can add other functions not just limited to download functionality in that file which are doing some calculations or providing customized output on the terminal.

I think `tui.py` will fulfill this purpose and more understandable than `helper.py` or `download.py`. 

going with `tui.py` then 👍

Reply:
I meant that if this function is not going to be called by anybody except the `download-...` set of commands, then `download.py` may be appropriate location. If it would be called by `download-...` and `get-...` etc commands, for example styling general output messages, then `tui.py` may be more appropriate location. I'm OK with both, as `cernopendata-client` will grow, we may actually be revamping things later. (E.g. `reana-dev` started as one single file, but as the number of commands grew, we separate each command set into its own file.)

Reply:
I agree and it sounds perfect. 👍